docs/features/backup_and_restore.md

```diff
@@ -6,7 +6,201 @@ hide_title: false
 
 ## Deployment Backup and Restore
 
-Most of your Cumulus deployment can be recovered by redeploying via Terraform. However, the Cumulus metadata including providers, collections, granules, rules, and executions that is stored in [DynamoDB](./data_in_dynamodb) can only be restored if backup was configured or enabled. If a deployment is lost, logs and Step Function executions in the AWS console will be irrecoverable.
+Most of your Cumulus deployment can be recovered by redeploying via Terraform.
+However, the Cumulus metadata including providers, collections, granules, rules,
+and executions that are stored in [DynamoDB](./data_in_dynamodb), and
+concurrently being written to the Core Postgres instance can only be
+restored if backup was properly configured or enabled. If a deployment is lost,
+logs and Step Function executions in the AWS console will be irrecoverable.
+
+## Postgres Database
+
+### Please note
+
+* Cumulus supports a "bring your own" Postgres instance approach, however
+  our reference implementation utilizes a serverless Aurora/RDS database - as
+  such this reference provides AWS RDS Aurora Serverless backup options.
+
+* As of Cumulus version 3, data storage is transitioning from the use of a
```

Comment:
This line is going to be out of date as we are currently looking at an unknown version for this release. Could we harvest it from somewhere in an automated way? Or is there some reference we can make to have docusaurus pick it up?

Reply:
If not, either leave it out or put in a placeholder version? It occurs to me that it would be nice to have something to pick up `TODO` style comments before commit.

Reply:
I'm not aware of any sane way to do that, but I can do a quick timebox on figuring out a way to do it.

Reply:
It looks like the best way to accomplish this is an upgrade to docusarus v2.

Reply:
ccccd3e42 for now.    When this was originally written we were not on a feature branch.